device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsIotHubConnection.java

```diff
@@ -172,14 +175,28 @@ public void registerMultiplexedDevice(DeviceClientConfig config)
         deviceClientConfigs.add(config);
     }
 
-    public void unregisterMultiplexedDevice(DeviceClientConfig config)
+    /**
+     * Asynchronously unregister a multiplexed device from an active multiplexed connection or synchronously unregister.
```

Comment:
nit: extra `.` at eol
q - this being async and returning a void, how does a user block on this or be notified when the async operation completes?

Reply:
user or us if the class is only internally used

Reply:
It's internal, but there is a callback defined elsewhere that allows for the AMQP layer to communicate to the transport agnostic layer when certain events happen (connectionLoss, connectionEstablished, and multiplexedDeviceRegistered/Unregistered)

Reply:
```suggestion
     * Asynchronously unregister a multiplexed device from an active multiplexed connection or synchronously unregister
```